### DISCUSSION TITLE
Is MetaData Civitai compatible?

### DISCUSSION THREAD

**Discussion by artofbeinghuman** [runew0lf/RuinedFooocus#112] [Q&A]
Is the metadata ruinedfooocus writes to the pngs compatible with civitai's automatic reading of all generation details (especially including loras)?

**Comment by runew0lf** [comment]
Nope unfortunately not, they keep telling us that theyre adding it, but at the moment they have been concentrating on getting comfyui working. Its a sore point for me as i hate typing in the descriptions manually!

**Comment by runew0lf** [comment]
IKt is now!!! Huzzah! they just added it
